connectors/tests/test_sync.py

```diff
@@ -436,37 +436,6 @@ async def test_connector_service_poll_large(mock_responses, patch_logger, set_en
     assert_re("Sync done: 1001 indexed, 0  deleted", patch_logger.logs)
 
 
-@pytest.mark.asyncio
-async def test_connector_service_poll_clear_error(
```

Comment:
What about the deleted tests? Was the functionality removed?

Reply:
I don't know exactly what it's trying to test, but the 2 deleted tests try to run `service.run()` twice.